agent/src/jvmMain/kotlin/com/epam/drill/agent/websocket/WsClient.kt

```diff
@@ -29,8 +29,8 @@ import com.epam.drill.common.message.MessageType
 
 object WsClient : WsClientReconnectHandler {
 
+    val endpoint = WsClientEndpoint(WsMessageHandler, this)

```

Comment:
Why not private?

Reply:
that field will be used by java-agent:
https://github.com/Drill4J/java-agent/pull/114/commits/5c40bf9ec4bb932f4b9eb80f38162e39bf06f547#diff-7df900a52df5b634c1ed5959c206a2665f975bc2a7f9819b8f8f756c20209b4eR49

Reply:
fixed